src/main.rs

```diff
@@ -1,16 +1,29 @@
-extern crate wasm_pack;
-
-extern crate indicatif;
-#[macro_use]
-extern crate quicli;
+extern crate failure;
 #[macro_use]
 extern crate human_panic;
+extern crate indicatif;
+extern crate structopt;
+extern crate wasm_pack;
 
-use quicli::prelude::*;
+use failure::Fail;
+use structopt::StructOpt;
 use wasm_pack::command::run_wasm_pack;
+use wasm_pack::error::Error;
 use wasm_pack::Cli;
 
-main!(|args: Cli, log_level: verbosity| {
+fn main() {
     setup_panic!();
+    if let Err(e) = run() {
+        eprintln!("{}", e);
+        for cause in e.causes().skip(1) {
+            eprintln!("Caused by: {}", cause);
```

Comment:
can you give some examples of what this error ends up looking like?

Reply:
This doesn't change functionality this is what quicli already does